powerauth-java-client-jaxws/src/main/resources/binding.xjb

```diff
@@ -0,0 +1,15 @@
+<jaxb:bindings xmlns:xs="http://www.w3.org/2001/XMLSchema"
+               xmlns:jaxb="http://java.sun.com/xml/ns/jaxb" xmlns:xjc="http://java.sun.com/xml/ns/jaxb/xjc"
+               jaxb:extensionBindingPrefixes="xjc" version="2.1">
+    <jaxb:globalBindings typesafeEnumMemberName="generateName" generateIsSetMethod="false"
+                         generateElementProperty="false">
+        <xjc:simple/>
+        <xjc:javaType adapter="io.getlime.powerauth.soap.adapter.DateTimeToLocalDateTime"
+                      name="java.time.LocalDateTime" xmlType="xs:dateTime"/>
+        <!--<xjc:javaType adapter="io.getlime.powerauth.soap.adapter.DateToLocalDate"
```

Comment:
I suggest to remove the commented out adapters, if there is no plan to use them.